src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
```

Comment:
Montserrat is not a system default font, so I highly suggest to install it via [Google Fonts](https://fonts.google.com/specimen/Montserrat) or other Webfont providers.

Read more about [Web Typography](https://internetingishard.com/html-and-css/web-typography/).